oas_docs/package-lock.json

```diff
@@ -1,4637 +0,0 @@
-{
```

Comment:
Same with this change?

Reply:
I think something corrupted the merge

Reply:
I didn’t delete this file, I'll try to bring it back